File: pgm/ABC_Java/ABC_334_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] BG = reader.readLine().split (" ");
        final String Bat = "Bat";
        final String Glove = "Glove";
        final int B = Integer.parseInt (BG[0]);
        final int G = Integer.parseInt (BG[1]);
        System.out.println (B > G ? Bat : Glove);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_334_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.util.StringTokenizer

// import kotlin.math.*

const val Bat: String = "Bat"
const val Glove: String = "Glove"
const val space: String = " "

fun main() {
    val (B: Int, G: Int) = readln().split(space).map { it.toInt() }
    if (B > G) {
        println(Bat)
    } else {
        println(Glove)
    }
    kotlin.system.exitProcess(0)
}